tracee-ebpf/tracee/userevents/system_info.go

```diff
@@ -0,0 +1,62 @@
+// Package userevents
+// Invoked tracee-ebpf events from user mode
+// This utility can prove itself useful to generate information needed by signatures that is not provided by normal
+// events in the kernel.
+// Because the events in the kernel are invoked by other programs behavior, we cannot anticipate which events will be
+// invoked and as a result what information will be extracted.
+// This is critical because tracee-rules is independent, and doesn't have to run on the same machine as tracee-ebpf.
+// This means that tracee-rules might lack basic information of the operating machine needed for some signatures.
+// By creating user mode events this information could be intentionally collected and passed to tracee-ebpf afterwards.
+package userevents
+
+import (
+	"github.com/aquasecurity/tracee/tracee-ebpf/external"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+)
+
+const INIT_PROC_NS_DIR = "/proc/1/ns"
+
+const INIT_NAMESPACES_KEY = "initNamespaces"
+
+// CreateSystemInfoEvent collect information of the running machine and create an event that includes this information.
+func CreateSystemInfoEvent() (external.Event, error) {
+	systemInfoArgs := fetchSystemInfo()
+	systemInfoEvent := external.Event{
+		Timestamp:   int(time.Now().UnixNano()),
```

Comment:
This is not in the format we use for Timesptamp

Reply:
It is weird because it is when I checked on the output

Reply:
![image](https://user-images.githubusercontent.com/87267148/131249135-48b7a1e3-81b1-45c5-b09a-a62b24aab95f.png)
You can see here that the first row is the system_info event, and the timestamp is identical to its following events